### PR TITLE
G ri d

### PR DESCRIPTION
**Pull request by asafpr**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).
This recipe installs a tool for a paper coming out, using DNA sequencing for measuring growth rate in a metagenomics sample

### CODE REVIEW

**Comment by asafpr** [comment]
The test fails on lofreq recipe which I didn't touch, and compiler 'c' which I added to meta.yaml file. I can't make the test pass

**Comment by asafpr** [comment]
@bioconda/core could you please review? My first recipe. Thanks

File: recipes/grid/build.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -eu -o pipefail
+
+outdir=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
+mkdir -p $outdir
+mkdir -p $PREFIX/bin
+
+cd $SRC_DIR
+
+cp -R * $outdir
```
Comment (bgruening) [inline recipes/grid/build.sh]: Do we need the test data? The package is big and I wonder if everything is need what you are copying here.

Comment (asafpr) [reply recipes/grid/build.sh]: It's not necessary but it's nice to have in the installation in order to test it. I can remove it.

**Pull request merged**